component/libfuse/libfuse.go

```diff
@@ -332,6 +332,14 @@ func (lf *Libfuse) Configure(_ bool) error {
 		return fmt.Errorf("%s config error %s", lf.Name(), err.Error())
 	}
 
+	// Disable libfuse logs if the mount is not running in foreground.
```

Comment:
We can not disable this. When we encounter any error during fuse init, we mount in foreground and enable to fuse logs. That's how we can debug what parameter we are sending to fuse, which it does not like or any specific error it can emit out. With this dsiabled there is no way to debug any fuse level logs.